bff-service/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const axios = require('axios').default;
+const cache = require('./middleware/cache');
+
+const app = express();
+
+app.use(express.json());
+
+app.all('/*', cache(120), (req, res) => {
```

Comment:
It is a mistake to cache all the requests to the BFF service.

Reply:
I'm caching only GET requests 
```
    if (req.method !== 'GET') {
        next();
    }
```
Please, look at bff-service/middleware/cache.js

Reply:
But what about user cart? Cart service should also process GET requests.

How https://xuatxu4bq6.execute-api.eu-west-1.amazonaws.com/dev/api/profile/cart can be accessed?